src/lib/jetlore.js

```diff
@@ -0,0 +1,18 @@
+/* @flow */
+let JL = {};
+!(function() { function o(o) { const t = this; t.access_token = o.cid, t.user_id = void 0 !== o.user_id && o.user_id != null && o.user_id !== '' && o.user_id.indexOf('/') < 0 ? o.user_id : 'undefined', t.feed_id = void 0 === o.feed_id || o.feed_id == '' ? 'any_feed' : o.feed_id, t.div = o.div, t.lang = o.lang, t.clicked = typeof o.clicked === 'function' ? o.clicked : function(o) {}, t.purchased = typeof o.purchased === 'function' ? o.purchased : function(o) {}, t.logToConsoleCookieName = 'Tracker_log_to_console', t.logToConsole = e._isConsoleCookieExists(t.logToConsoleCookieName), t.setCookies = void 0 !== o.setCookies && o.setCookies != null && o.setCookies, t.autoPageView = void 0 !== o.autoPageView && typeof o.autoPageView === 'boolean' && o.autoPageView, t.log(`Tracking initialized with parameters\n\t - access token:${ t.access_token  }\n\t - user id     :${  t.user_id  }\n\t - feed id     :${  t.feed_id  }\n\t - div         :${  t.div  }\n\t - lang        :${  t.lang }`), t.autoPageView && t.page_view(); }JL = { tracking(e) { JL.tracker = new o(e); } }, o.prototype.track = function(o) { const e = this,
```

Comment:
This is going to de-opt the webpack build, since it has side effects at the module scope. Can we wrap this in a `getJetlore()` or something if we're going to inline it?